app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardPresenter.kt

```diff
@@ -0,0 +1,28 @@
+package org.oppia.app.topic.conceptcard
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import kotlinx.android.synthetic.main.conceptcard_fragment.view.*
+import org.oppia.app.R
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.viewmodel.ViewModelProvider
+import javax.inject.Inject
+
+@FragmentScope
+class ConceptCardPresenter @Inject constructor(
+  private val fragment: Fragment,
+  private val viewModelProvider: ViewModelProvider<ConceptCardViewModel>
+  ){
```

Comment:
Nit: there shouldn't be spacing before the ``){`` since it's finishing the constructor block from line 15 that's also not horizontally spaced (similar to how closing braces have the same alignment as their opening brace line).